hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/server/TestSCMSecurityProtocolServer.java

```diff
@@ -32,10 +39,19 @@
 
   @Rule
   public Timeout timeout = new Timeout(1000 * 20);
+  private static int scmRpcSecurePort;
+
+  @BeforeClass
+  public static void setupClass() throws Exception {
+    scmRpcSecurePort = new ServerSocket(0).getLocalPort();
+  }
 
   @Before
   public void setUp() throws Exception {
     config = new OzoneConfiguration();
+    config.set(OZONE_SCM_SECURITY_SERVICE_ADDRESS_KEY,
+        StringUtils.join(OZONE_SCM_SECURITY_SERVICE_BIND_HOST_DEFAULT,
+            ":", String.valueOf(scmRpcSecurePort)));
```

Comment:
Here we can directly specify OZONE_SCM_SECURITY_SERVICE_BIND_HOST_DEFAULT:0
is there any reason for doing this way?

Reply:
@bharatviswa504 No specific reason. I just thought StringUtils is cleaner. I can do a simple concatenation.

Reply:
My comment is not related to the usage of StringUtils, we can directly use OZONE_SCM_SECURITY_SERVICE_BIND_HOST_DEFAULT:0, instead of getting from scmRpcSecurePort, which we get from new ServerSocket(0).getLocalPort(); In this way during server start, it will choose available free random port.